dataprofiler/profilers/profile_builder.py

```diff
@@ -834,6 +862,30 @@ def report(self, report_options: dict = None) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseProfiler:
```

Comment:
is load_from_dict for baseProfiler supposed to be in this PR

Reply:
will fix in second to next push